### PR TITLE
v10

### PR DESCRIPTION
**Pull request by austinkelleher**
Releases #939 #940

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,12 +27,12 @@
     "@typescript-eslint/eslint-plugin": "^6.2.1",
     "@typescript-eslint/parser": "^6.2.1",
     "eslint": "^8.46.0",
-    "eslint-config-prettier": "^8.9.0",
+    "eslint-config-prettier": "^9.0.0",
```
Comment (austinkelleher) [inline package.json:30]: See below for why I bumped this too. Ended up being unrelated, but it's a good thing to update anyway. 😁

File: package.json [hunk 2]
```diff
@@ -27,12 +27,12 @@
...
     "eslint-plugin-jest": "^27.2.3",
     "husky": "^4.2.5",
     "jest": "^29.6.1",
     "jest-extended": "^4.0.0",
-    "lerna": "^6.0.0",
+    "lerna": "^7.1.4",
```
Comment (austinkelleher) [inline package.json:35]: Needed to bump this due to a compatibility issue between Lerna 6 and Prettier 3. Here was the fix in Lerna: https://github.com/lerna/lerna/pull/3766/files#diff-f7957c0c5c84c95434097a432a049ba780cdccfd5db0a156de23911e6f5ff776R36

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,6 +9,15 @@ and this project adheres to
...
 ## Unreleased
 
+## 10.0.0 - 2023-08-03
+
+- Bump TypeScript, ESLint, and Prettier to next major versions. Integration
+  projects may encounter breaking changes when they adopt these new versions.
+  For example, Prettier changed some default formatting. To fix the Prettier
+  formatting, you may run the following command:
+
+  `yarn:format` or `prettier --write \"**/*.{ts,js,json,css,md,yml}\"`
```
Comment (VDubber) [inline CHANGELOG.md:19]: `yarn:format` -> `yarn format`

Comment (austinkelleher) [reply CHANGELOG.md:19]: Good catch. Will fix in follow-up PR to avoid having to move tags around :D

**Review by Nick-NCSU — APPROVED** [review]

**Pull request merged**